src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -6344,6 +6586,108 @@ public void Property_BlockBodyToExpressionBody2()
+            });
+        }
+
+        [Fact, WorkItem(17681, "https://github.com/dotnet/roslyn/issues/17681")]
+        public void Property_GetterExpressionBodyToExpressionBody()
+        {
+            var src1 = "class C { int P { get => 2; } }";
+            var src2 = "class C { int P => 1; }";
+
+            var edits = GetTopEdits(src1, src2);
+            edits.VerifyEdits(
+                "Update [int P { get => 2; }]@10 -> [int P => 1;]@10",
+                "Delete [{ get => 2; }]@16",
+                "Delete [get => 2;]@18");
+
+            edits.VerifySemantics(ActiveStatementsDescription.Empty, new[]
+            {
+                SemanticEdit(SemanticEditKind.Update, c => c.GetMember("C.get_P"), preserveLocalVariables: false),
+            });
+        }
+
+        [Fact, WorkItem(17681, "https://github.com/dotnet/roslyn/issues/17681")]
+        public void Property_GetterBlockBodyToGetterExpressionBody()
+        {
+            var src1 = "class C { int P { get { return 2; } } }";
+            var src2 = "class C { int P { get => 2; } }";
+            
+            var edits = GetTopEdits(src1, src2);
+
+            edits.VerifyEdits("Update [get { return 2; }]@18 -> [get => 2;]@18");
+
+            edits.VerifySemantics(ActiveStatementsDescription.Empty, new[]
+            {
+                SemanticEdit(SemanticEditKind.Update, c => c.GetMember("C.get_P"), preserveLocalVariables: false),
+            });
+        }
+
+        [Fact, WorkItem(17681, "https://github.com/dotnet/roslyn/issues/17681")]
+        public void Property_GetterExpressionBodyToGetterBlockBody()
+        {
+            var src1 = "class C { int P { get => 2; } }";
+            var src2 = "class C { int P { get { return 2; } } }";
+
+            var edits = GetTopEdits(src1, src2);
+
+            edits.VerifyEdits("Update [get => 2;]@18 -> [get { return 2; }]@18");
+
+            edits.VerifySemantics(ActiveStatementsDescription.Empty, new[]
+            {
+                SemanticEdit(SemanticEditKind.Update, c => c.GetMember("C.get_P"), preserveLocalVariables: false)
+            });
+        }
+
+        [Fact, WorkItem(17681, "https://github.com/dotnet/roslyn/issues/17681")]
+        public void Property_GetterBlockBodyWithSetterToGetterExpressionBodyWithSetter()
+        {
+            var src1 = "class C { int P { get => 2; set { Console.WriteLine(0); } } }";
+            var src2 = "class C { int P { get { return 2; } set { Console.WriteLine(0); } } }";
+
+            var edits = GetTopEdits(src1, src2);
+
+            edits.VerifyEdits("Update [get => 2;]@18 -> [get { return 2; }]@18");
+
+            edits.VerifySemantics(ActiveStatementsDescription.Empty, new[]
+            {
+                SemanticEdit(SemanticEditKind.Update, c => c.GetMember("C.get_P"), preserveLocalVariables: false),
+                SemanticEdit(SemanticEditKind.Update, c => c.GetMember("C.set_P"), preserveLocalVariables: false)
```

Comment:
>SemanticEdit(SemanticEditKind.Update, c => c.GetMember("C.set_P"), preserveLocalVariables: false) [](start = 16, length = 97)

I guess there is an update because the column changed.
Can we change the test to avoid that? Just add some spaces before "set" keyword in src1 to make sure the position of the setter doesn't change.